net/Socket.hpp

```diff
@@ -624,7 +624,7 @@ class SocketPoll
             }
         }
 #endif
-        wakeup();
+        speedwakeup();
```

Comment:
Is there a need to rename the method ? and if so, why - are there usefully two variants that do different things ?